benchmark/Streamly/Benchmark/Prelude/Rate.hs

```diff
@@ -51,48 +52,48 @@ constRate value rate_ = S.constRate rate_ . sourceUnfoldrM value
 o_1_space_async :: Int -> [Benchmark]
 o_1_space_async value =
     [ bgroup
-          "asyncly"
-          [ bgroup
-                "avgRate"
-                -- benchIO "unfoldr" $ toNull fromAsync
-                -- benchIOSrc fromAsync "unfoldrM" (sourceUnfoldrM value)
-                [ benchIOSrc fromAsync "Nothing" $ rateNothing value
-                , benchIOSrc fromAsync "1M" $ avgRate value 1000000
-                , benchIOSrc fromAsync "3M" $ avgRate value 3000000
-                , benchIOSrc fromAsync "10M/maxThreads1"
-                      $ avgRateThreads1 value 10000000
-                , benchIOSrc fromAsync "10M" $ avgRate value 10000000
-                , benchIOSrc fromAsync "20M" $ avgRate value 20000000
-                ]
-          , bgroup
-                "minRate"
-                [ benchIOSrc fromAsync "1M" $ minRate value 1000000
-                , benchIOSrc fromAsync "10M" $ minRate value 10000000
-                , benchIOSrc fromAsync "20M" $ minRate value 20000000
-                ]
-          , bgroup
-                "maxRate"
-                [ -- benchIOSrc fromAsync "10K" $ maxRate value 10000
-                  benchIOSrc fromAsync "10M" $ maxRate value 10000000
-                ]
-          , bgroup
-                "constRate"
-                [ -- benchIOSrc fromAsync "10K" $ constRate value 10000
-                  benchIOSrc fromAsync "1M" $ constRate value 1000000
-                , benchIOSrc fromAsync "10M" $ constRate value 10000000
-                ]
-          ]
+        "asyncly"
+        [ bgroup
+            "avgRate"
+            -- benchIO "unfoldr" $ toNull fromAsync
+            -- benchIOSrc fromAsync "unfoldrM" (sourceUnfoldrM value)
+            [ benchIOSrc fromAsync "Nothing" $ rateNothing value
+            , benchIOSrc fromAsync "1M" $ avgRate value 1000000
+            , benchIOSrc fromAsync "3M" $ avgRate value 3000000
+            , benchIOSrc fromAsync "10M/maxThreads1"
+                $ avgRateThreads1 value 10000000
+            , benchIOSrc fromAsync "10M" $ avgRate value 10000000
+            , benchIOSrc fromAsync "20M" $ avgRate value 20000000
+            ]
+        , bgroup
+            "minRate"
+            [ benchIOSrc fromAsync "1M" $ minRate value 1000000
+            , benchIOSrc fromAsync "10M" $ minRate value 10000000
+            , benchIOSrc fromAsync "20M" $ minRate value 20000000
+            ]
+        , bgroup
+            "maxRate"
+            [ -- benchIOSrc fromAsync "10K" $ maxRate value 10000
+            benchIOSrc fromAsync "10M" $ maxRate value 10000000
+            ]
+        , bgroup
+            "constRate"
+            [ -- benchIOSrc fromAsync "10K" $ constRate value 10000
+            benchIOSrc fromAsync "1M" $ constRate value 1000000
+            , benchIOSrc fromAsync "10M" $ constRate value 10000000
+            ]
+        ]
```

Comment:
These are unrelated formatting changes, can be reverted or made in a separate commit.